### DISCUSSION TITLE
atoms 관련 파일을 src/recoil 내부로 이동시키는 것에 대해 어떻게 생각하시나요?

### DISCUSSION THREAD

**Discussion by minjongbaek** [prgrms-web-devcourse/Team-Gaerval-Dadok-FE#268] [General]
atoms 관련 파일을 src/recoil 내부로 이동시키는 것에 대해 어떻게 생각하시나요?

지금은 hooks 안에 위치하고 있는데 범용성을 생각한다면 src 바로 아래에 위치하는 것이 괜찮지 않을까 하는 생각이 있습니다~

**Comment by gxxrxn** [comment]
오.. 범용적으로 사용되는 atom이라면 밖으로 빼도 좋을 것 같아요!!

**Reply by minjongbaek** [reply]
앗 저는 액세스 토큰을 포함한 모든 atom을 바깥으로 빼고 싶었어요..!

전역으로 관리하는 모든 상태를 한 눈에 파악할 수 있으면 좋을 것 같다고 생각해서요!
